docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -0,0 +1,248 @@
+MLflow Tracking Quickstart
+==========================
+
+Welcome to MLflow! 
+
+The purpose of this quickstart is to provide a quick guide to the most essential core APIs of MLflow Tracking. 
+Specifically, those that enable the logging, registering, and loading of a model for inference. 
+
+.. note::
+    For a more in-depth and tutorial-based approach (if that is your style), please see the 
+    `Getting Started with MLflow <../logging-first-model/index.html>`_ tutorial. We recommend that you start here first, though, as this quickstart 
+    uses the most common and frequently-used APIs for MLflow Tracking and serves as a good foundation for the other tutorials in the documentation. 
+
+What you will learn
+-------------------
+
+In just a few minutes of following along with this quickstart, you will learn:
+
+* How to **log** parameters, metrics, and a model
+* The basics of the **MLflow fluent API**
+* How to **register** a model during logging 
+* How to navigate to a model in the **MLflow UI** 
+* How to **load** a logged model for inference 
+
+If you would like to see this quickstart in a purely notebook format, we have a downloadable and viewable notebook-only version of this quickstart:
+
+.. raw:: html
+
+    <a href="notebooks/index.html" class="download-btn">View the Notebook</a>
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    notebooks/index
+
+
+Step 1 - Get MLflow
+-------------------
+
+MLflow is available on PyPI. If you don't already have it installed on your system, you can install it with:
+
+.. code-section::
+    .. code-block:: bash
+        :name: download-mlflow
+
+        pip install mlflow
+
+Step 2 - Start a Tracking Server
+--------------------------------
+
+We're going to start a local MLflow Tracking Server, which we will connect to for logging our data for this quickstart.
+From a terminal, run:
+
+.. code-section::
+    .. code-block:: bash
+        :name: tracking-server-start
+
+        mlflow server --host 127.0.0.1 --port 8080
```

Comment:
```suggestion
.. code-block:: bash
    :name: tracking-server-start

    mlflow server --host 127.0.0.1 --port 8080
```

Can we remove `code-section:`?

Reply:
All code blocks throughout the docs are going to be converted to use language-specific indicators to match the style of Databricks docs. The code-section formatting is a consistent formatting going forward which will make much more sense as examples for R are added (so that code blocks are selectable to change the displayed code within the docs).

Reply:
sounds good :)